cmd/tmpfs/tmpfs.go

```diff
@@ -345,10 +345,11 @@ func main() {
 		arch.DumpArchive()
 	}
 
-	// Bind and listen on the socket.
-	listener, err := net.Listen(*network, *netaddr)
+	netaddr = flag.String("addr", ":5641", "Network address")
```

Comment:
Why move this flag down here?  Is it even used any more?

Reply:
it's gone :-)